cloud/maintenance.md

```diff
@@ -107,7 +109,28 @@ creating a fork of your service, see the
 
 </procedure>
 
+### Best practices for upgrading
+Follow some best practices for a smooth upgrade experience.
+
+#### Create a fork
+Fork your database before you upgrade. This allows you to:
+*   Try out the upgrade on the fork before running it on your production system.
+    This gives you a good idea of what happens during the upgrade, and how long it
+    might take.
+*   Keep a copy of your old database version if you're worried about losing it
```

Comment:
Creating a fork and creating a copy of the old database are two different procedures. I would list them as three bullet points under best practices with the links to relevant pages to avoid redundancy:
* Create a fork 
* Back up the database
* Enable replication